src/v3/src/transformer/securityQuestion/securityQuestionEnroll.ts

```diff
@@ -114,17 +108,6 @@ export const transformSecurityQuestionEnroll: IdxStepTransformer = (transaction,
     },
   };
 
-  const customQuestionKeyElement: FieldElement = {
-    type: 'Control',
-    name: QUESTION_KEY_INPUT_NAME,
-    options: {
-      inputMeta: {
-        ...(customQuestionOptions?.[0].value as Input[]).find(({ name }) => name === 'questionKey'),
-        name: QUESTION_KEY_INPUT_NAME,
-      },
-    },
-  };
```

Comment:
Curious, what was this used for previously? I see it just removed but nothing replacing it.

Reply:
ohh, seems like this is an immutable element in uischema, which should be excluded and handled during submission. I think this is the element that causes warnings in jest.